app/src/main/java/com/waseefakhtar/doseapp/DoseApp.kt

```diff
@@ -58,6 +65,9 @@ fun DoseApp() {
             modifier = Modifier.fillMaxSize(),
             color = MaterialTheme.colorScheme.background
         ) {
+            // For the snackbar
+            var openMySnackbar by remember { mutableStateOf(false) }
+            var snackBarMessage by remember { mutableStateOf("") }
```

Comment:
I'd also like the naming to be similar to Material Design's naming:

```suggestion
            var snackbarMessage by remember { mutableStateOf("") }
```

So we can replace `snackBar` with `snackbar` in all other places too. :)